src/shapes/text.class.js

```diff
@@ -506,7 +458,7 @@
         return;
       }
 
-      var lineWidth = ctx.measureText(line).width,
+      var lineWidth = this._getLineWidth(ctx, i),
```

Comment:
@jafferhaider 
my foopa!
Can you change in this line i with LineIndex, i and check problem goes away? If yes i will do the PR to fix it.